pkg/cluster/cluster_config.go

```diff
@@ -53,16 +53,25 @@ func IsNotReservedNamespace(ns *corev1.Namespace) bool {
 // GetClusterServiceVersion retries the clusterserviceversions available in the operator namespace.
```

Comment:
I am not quite sure this godoc reflects the actual behaviour of this function. Maybe worth updating.